css/custom.css

```diff
@@ -103,6 +101,10 @@ section {
     margin-bottom:30px;
 }
 
+.section-head{
+    margin-top: 50px;
```

Comment:
This is going to affect both the mobile and desktop view. Please make sure this only on desktop view. :)